src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -203,6 +216,10 @@ public void onBuildComplete(AbstractBuild<?, ?> build,
             }
         }
 
+        if((showMatrixStatus && build.getProject() instanceof MatrixProject)){
```

Comment:
However, it should be !(build.getProject() intanceof MatrixProject) since children jobs return false when checking if its an instance of a matrix project, but the parent job will return true, so I will fix this up as well as the parenthesis, and accounting for the null pointers on my next commit